README.md

```diff
@@ -738,7 +738,7 @@ This decorator provides support for letting you customize its behaviour to easil
     // sync with the server, without ever
     // interrupting the end user
     class MyApp extends Component {}
-    codePush()(MyApp);
+    codePush(MyApp);
```

Comment:
Oh crap, I made a mistake earlier - these should be `MyApp = codePush(MyApp)`, here and below